src/styles/blocks/card.scss

```diff
@@ -0,0 +1,62 @@
+.card {
+  box-sizing: border-box;
+  position: relative;
+  text-align: justify;
+  width: 200px;
+  height: 407px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+  transition: all 0.3s linear;
+}
+
+.card__product {
+  display: block;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $standart-black-color;
+  text-decoration: none;
+}
+
+.card__code {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding-inline: $standart-inline-pad;
+  padding-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  color: #616070;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-inline: 16px;
+  height: 40px;
+  border: 1px solid $grey-star-color;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  text-decoration: none;
+  color: #fff;
+  background-color: $blue-hover;
+  cursor: pointer;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  border: 1px solid $blue-hover;
+  border-radius: 5px;
+  color: $blue-hover;
+}
+
+.card:hover {
+  transform: scale(1.2);
```

Comment:
The hover effect on the card scales it to 1.2 times its original size. This might cause layout issues if the surrounding elements are not prepared for this change. Consider testing this effect in the context of your layout.